wpe/src/main/java/com/wpe/wpe/Browser.java

```diff
@@ -537,4 +537,12 @@ public void deleteInputMethodContent(@NonNull WPEView wpeView, int offset)
         }
         m_pages.get(wpeView).deleteInputMethodContent(offset);
     }
+
+    public void requestExitFullscreenMode(@NonNull WPEView wpeView)
+    {
+        if (m_pages == null || !m_pages.containsKey(wpeView)) {
+            return;
+        }
```

Comment:
{} are not useful

Reply:
follows the convention that was already used elsewhere in the file for similar code. We can reformat whole file later with consistent convention